### PR TITLE
Payroll employee list pagination

### LINKED ISSUE #27: Employee list pagination

### PR DESCRIPTION
**Pull request by sgobotta**
This PR provides:

+ Pagination feature into the Table component
+ 2 more accounts in the PayrollKit contract

closes #27

### CODE REVIEW

**Comment by josx** [comment]
There is an error on linting the code.
```
standard: Use JavaScript Standard Style (https://standardjs.com)
/home/rof/src/github.com/protofire/aragon-apps/future-apps/payroll/app/src/store/index.js:8:10: 'date' is defined but never used.
```

I will fix it

File: future-apps/payroll/app/src/components/Table/Table.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Card } from '@aragon/ui'
 import { Button, Table as BaseTable, Text } from '@aragon/ui'
```
Comment (josx) [inline future-apps/payroll/app/src/components/Table/Table.js]: would be better to import all aragon/ui components in just one line

File: future-apps/payroll/app/src/components/Table/Table.js [hunk 2]
```diff
@@ -8,10 +9,29 @@ import TableRow from './TableRow'
 import Panel from '../Panel/Panel'
 import { sort, SORT_DIRECTION } from '../../utils/sorting'
 
+const footerStyles = {
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/components/Table/Table.js]: I think we should use styled-component for this

File: future-apps/payroll/app/src/components/Table/Table.js [hunk 3]
```diff
@@ -39,8 +58,92 @@ class Table extends React.Component {
...
+    const paginatedData = data.slice(firstDataIndex, lastDataIndex)
+    const emptyRows = rowsPerPage - paginatedData.length
+    const totalPages = Math.ceil(data.length / rowsPerPage)
+
+    if (paginatedData.length === 0) this.setState({ currentPage: 1 })
+
+    return {
+      emptyRows,
+      paginatedData,
+      totalPages
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/components/Table/Table.js]: I think thi is not being used, we could remove it or use it maybe `pageNumbers ` inside `renderPageButtons`

File: future-apps/payroll/app/src/components/Table/Table.js [hunk 4]
```diff
@@ -39,8 +58,92 @@ class Table extends React.Component {
...
+          style={isFirstPage ? inactiveNavigationButton : activeNavigationButton }
+          onClick={this.handlePageChange(currentPage - 1)}
+        >
+          { navigationPreviousText }
+        </Button>
+        {pageNumbers.map((pageNumber) => {
+          const isCurrentPageButton = pageNumber === currentPage
+          const name = `pagination-button-${pageNumber}`
+          return (
+            <Button
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/components/Table/Table.js]: We could have a styled-component for this

File: future-apps/payroll/app/src/components/Table/Table.js [hunk 5]
```diff
@@ -39,8 +58,92 @@ class Table extends React.Component {
...
+              name={name}
+              mode={isCurrentPageButton ? 'strong' : ''}
+              style={isCurrentPageButton ? inactivePageButton : activePageButton}
+              onClick={this.handlePageChange(pageNumber)}
+              >
+              {pageNumber}
+            </Button>
+          )
+        })}
+        <Button
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/components/Table/Table.js]: We could have a styled-component for this

File: future-apps/payroll/app/src/components/Table/Table.js [hunk 6]
```diff
@@ -83,37 +189,61 @@ class Table extends React.Component {
...
+                  />
+              )
+            })}
+          </TableRow>
+        ))}
+        {this.renderEmptyRows(emptyRows)}
+      </React.Fragment>
+    )
+
+    const footer = (
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/components/Table/Table.js]: We could have a styled-component for this

File: future-apps/payroll/app/src/components/Table/Table.js (hunk 3 above)
Comment (sgobotta) [reply future-apps/payroll/app/src/components/Table/Table.js]: Yes, I'm taking it out.

File: future-apps/payroll/app/src/components/Table/Table.js (hunk 6 above)
Comment (sgobotta) [reply future-apps/payroll/app/src/components/Table/Table.js]: Moved footer content to a styled component

File: future-apps/payroll/app/src/components/Table/Table.js (hunk 4 above)
Comment (sgobotta) [reply future-apps/payroll/app/src/components/Table/Table.js]: Page buttons were refactored

File: future-apps/payroll/app/src/components/Table/Table.js (hunk 2 above)
Comment (sgobotta) [reply future-apps/payroll/app/src/components/Table/Table.js]: The footer styles were taken out to a single StyledButton styled component

**Pull request merged**